dlp/snippets/deid.py

```diff
@@ -1282,6 +1399,36 @@ def deidentify_with_exception_list(
         help="The list of strings to ignore matches against.",
     )
 
+    table_row_suppress_parser = subparsers.add_parser(
+        "deid_table_row_suppress",
+        help="De-identify sensitive data in a table by suppressing "
+             "entire row/s based on a condition.",
+    )
+    table_row_suppress_parser.add_argument(
+        "project",
+        help="The Google Cloud project id to use as a parent resource.",
+    )
+    table_row_suppress_parser.add_argument(
+        "table_data",
+        help="Json string representing table data",
+    )
+    table_row_suppress_parser.add_argument(
+        "--condition_field",
+        help="A table Field within the record this condition is evaluated "
+             "against.",
+    )
+    table_row_suppress_parser.add_argument(
+        "--condition_operator",
```

Comment:
Some of the arguments are present in another PR as well? Needs resolving conflicts.

Reply:
Arguments are for different functions.  there would be no conflict.